packages/EasyEntityChange/tests/Stubs/EventDispatcherStub.php

```diff
@@ -47,4 +47,12 @@ public function getDispatched(): array
     {
         return $this->dispatched;
     }
+
+    /**
+     * @return mixed[]
+     */
+    public function getReturns(): array
+    {
+        return $this->returns;
+    }
```

Comment:
This is to make `$returns` property actually used

Reply:
@TomasVotruba thank you for the PR, this entire package is gonna disappear because not used anymore 😄